config/deploy.js

```diff
@@ -28,6 +29,12 @@ module.exports = function(deployTarget) {
     ENV.redis.url = process.env.ORG_PRODUCTION_REDIS_URL;
   }
 
+  if (deployTarget === 'org-staging-pull-request') {
+    ENV.s3.bucket = 'travis-web-production-next';
+    ENV.redis.url = process.env.ORG_PRODUCTION_REDIS_URL;
```

Comment:
This instance of Redis stores the `index.html` file that links to the JS and CSS assets a particular deployment of `travis-web` is composed of. This is known as the “lightning” strategy of deployment for Ember CLI applications.

Since these are all ephemeral deployments produced by `travis-web` pull requests, it seems harmless to me to use the same instance of Redis. It’s really just a collection of HTML files.

More on this below.